src/vm/hints/hint_utils.rs

```diff
@@ -36,11 +36,11 @@ pub fn insert_list_into_scope(exec_scopes: &mut ExecutionScopes, name: &str, lis
 
 //Returns the value in the current execution scope that matches the name and is of type BigInt
 pub fn get_int_from_scope(
-    vm: &mut VirtualMachine,
+    exec_scopes: &mut ExecutionScopes,
```

Comment:
There doesn't seem to be a need for it to be mutable if it returns a copy of the value.

Reply:
Included in refactor